README.md

```diff
@@ -1,5 +1,7 @@
 # Federation API
 
+Please refer to our [**official documentation**](https://neurobagel.org/overview/) for more information on how to use the federation API.
```

Comment:
Since we don't have any official documentation on the federation API yet, wdyt about something like this for now?

```suggestion
Please refer to our [**official documentation**](https://neurobagel.org/) for more information on Neurobagel federation.
```